Sources/View/UICollectionView+Reusable.swift

```diff
@@ -18,8 +18,8 @@ public extension UICollectionView {
 
    - seealso: `registerNib(_:,forCellWithReuseIdentifier:)`
```

Comment:
Docs for this and `UITableView` extension still refers to Swift 2.x

-> `register(_:forCellReuseIdentifier:)`